docs/operations/hotplug_interfaces.md

```diff
@@ -100,6 +100,34 @@ You can now check the VMI status for the presence of this new interface:
 kubectl get vmi vmi-fedora -ojsonpath="{ @.status.interfaces }"
 ```
 
+### Migration based hotplug
+In case your cluster doesn't meet the requirements, it's possible to hotplug an interface by migrating the VM.
+The actual attachment wont take place immediately, the new interface will be ready once the migration is completed.
+
+#### Add new interface:
+```bash
+virtctl addinterface vmi-fedora --network-attachment-definition-name new-fancy-net --name dyniface1
+```
+At this point the new interface is added to the spec but isn't attached to the VM. 
+
+#### Migrate the VM:
```

Comment:
Would it make sense to link to the migration docs instead ? You're repeating a documented process.

Reply:
I think that having all the commands right in front of you makes it easier to understand the flow (instead of jumping to another page).
How about keeping the migration-with-virtctl and add a link to the title?
If you think its also redundant I will remove it all (and keep the link only)

Reply:
Your call. I don't mind either way.